Makefile

```diff
@@ -22,12 +22,12 @@ build: operator register support
 
 .PHONY: operator
 operator:
-	go build -ldflags '$(LDFLAGS)' -o build/elemental-operator $(ROOT_DIR)/cmd/operator
+	CGO_ENABLED=0 go build -ldflags '$(LDFLAGS)' -o build/elemental-operator $(ROOT_DIR)/cmd/operator
 
 
 .PHONY: register
 register:
-	go build -ldflags '$(LDFLAGS)' -o build/elemental-register $(ROOT_DIR)/cmd/register
+	CGO_ENABLED=1 go build -ldflags '$(LDFLAGS)' -o build/elemental-register $(ROOT_DIR)/cmd/register
```

Comment:
can you also add it to the support build down below so we are explicit?

Reply:
Done!